core/modules/set_metadata.h

```diff
@@ -34,11 +34,15 @@
 #include "../module.h"
 #include "../pb/module_msg.pb.h"
 
-using bess::metadata::mt_offset_t;
 using bess::metadata::kMetadataAttrMaxSize;
+using bess::metadata::mt_offset_t;
 
-typedef struct { uint8_t bytes[kMetadataAttrMaxSize]; } value_t;
-typedef struct { uint8_t bytes[kMetadataAttrMaxSize]; } mask_t;
+typedef struct {
+  uint8_t bytes[kMetadataAttrMaxSize];
+} value_t;
+typedef struct {
+  uint8_t bytes[kMetadataAttrMaxSize];
+} mask_t;
```

Comment:
(Huh, did clang-format change?)

Reply:
oh, boo it did. didn't even noticed when i scanned the diff earlier